app/controllers/sales_controller.rb

```diff
@@ -3,14 +3,20 @@ class SalesController < ApplicationController
     load_and_authorize_resource
 
     def index
-      @sales = @sales.in_progress.page(params[:page])
+      if params[:cart]
+        @sales = @sales.in_progress.page(params[:page])
+        render 'cart'
+      else
+        @sales = @sales.completed.page(params[:page])
+        render 'history'
+      end
```

Comment:
I don't nkow what is the best practice here but it sounds like 2 separate actions

Reply:
I used resources to route this controller so I didn't want to change it to use an extra action

Reply:
not like the most trustful stackoverflow question to follow with only 3 upvotes but appreciate the effort.